agents/src/tokenize/basic/hyphenator.ts

```diff
@@ -51,7 +51,7 @@ class Hyphenator {
     // if the word is an exception, get the stored points
     let points: number[];
     if (this.exceptions[word.toLowerCase()]) {
-      points = this.exceptions[word.toLowerCase()];
+      points = this.exceptions[word.toLowerCase()]!;
```

Comment:
maybe initialize `points` with this and execute the `else` clause if points is undefined?